src/services/workflow.service.js

```diff
@@ -54,6 +54,7 @@ class SubscriptionWorkflowService extends GQuery {
    */
   constructor (httpUrl, subscriptionClient) {
     super()
+    this.subscriptionClient = subscriptionClient
```

Comment:
Needed this so that we could pass the same instance to the GraphiQL fetcher.

Tried to access the object from the instance of `$workflowService`, but it's hidden deep in another object. So the easiest approach was this one.